components/dashboard/src/Menu.tsx

```diff
@@ -315,35 +314,50 @@ export default function Menu() {
                     </ContextMenu>
                 </div>
-                {projectSlug && (
-                    <div className="flex h-full rounded-lg hover:bg-gray-100 dark:hover:bg-gray-800 px-2 py-1">
-                        <Link to={`${teamOrUserSlug}/${projectSlug}${prebuildId ? "/prebuilds" : ""}`}>
-                            <span className="text-base text-gray-600 dark:text-gray-400 font-semibold">
-                                {project?.name}
-                            </span>
-                        </Link>
-                    </div>
+                {projectSlug && !prebuildId && (
+                    <Link to={`${teamOrUserSlug}/${projectSlug}${prebuildId ? "/prebuilds" : ""}`}>
+                        <span className=" flex h-full text-base text-gray-50 bg-gray-800 dark:bg-gray-50 dark:text-gray-900 font-semibold ml-2 px-3 py-1 rounded-2xl border-gray-100">
+                            {project?.name}
+                        </span>
+                    </Link>
                 )}
                 {prebuildId && (
-                    <div className="flex h-full ml-2 py-1">
-                        <img alt="" className="mr-3 filter-grayscale m-auto transform -rotate-90" src={CaretDown} />
-                        <span className="text-base text-gray-600 dark:text-gray-400 font-semibold">{prebuildId}</span>
+                    <Link to={`${teamOrUserSlug}/${projectSlug}${prebuildId ? "/prebuilds" : ""}`}>
+                        <span className=" flex h-full text-base text-gray-500 bg-gray-50 hover:bg-gray-100 dark:text-gray-400 dark:bg-gray-800 dark:hover:bg-gray-700 font-semibold ml-2 px-3 py-1 rounded-2xl border-gray-100">
+                            {project?.name}
+                        </span>
+                    </Link>
+                )}
+                {prebuildId && (
+                    <div className="flex ml-2">
+                        <div className="flex pl-0 pr-1 py-1.5">
+                            <svg width="20" height="20" fill="none" xmlns="http://www.w3.org/2000/svg">
+                                <path
+                                    fill-rule="evenodd"
+                                    clip-rule="evenodd"
+                                    d="M7.293 14.707a1 1 0 0 1 0-1.414L10.586 10 7.293 6.707a1 1 0 1 1 1.414-1.414l4 4a1 1 0 0 1 0 1.414l-4 4a1 1 0 0 1-1.414 0Z"
+                                    fill="#78716C"
+                                />
+                            </svg>
+                        </div>
+                        <Link to={`${teamOrUserSlug}/${projectSlug}/${prebuildId}`}>
+                            <span className="flex h-full text-base text-gray-50 bg-gray-800 dark:bg-gray-50 dark:text-gray-900 font-semibold px-3 py-1 rounded-2xl border-gray-100">
+                                {prebuildId.substring(0, 8).trimEnd()}
+                            </span>
+                        </Link>
                     </div>
                 )}
```

Comment:
**thought**: Not sure if this could be better handled. Sorry for the potentially duplicate code here, but it helped push this forward, @AlexTugarev! 😬